Buddies/CreateActivity/CreateActivityVC.swift

```diff
@@ -337,6 +343,7 @@ extension CreateActivityVC: MKLocalSearchCompleterDelegate {
     
     private func completerDidUpdateResults(completer: MKLocalSearchCompleter) {
         searchResults = completer.results
+        
     }
     
```

Comment:
Do we save the nice text to firebase yet?

Reply:
ye we do